### DISCUSSION TITLE
Como puedo ser co-author de un commit.

### DISCUSSION THREAD

**Discussion by VHAA1718** [community/community#20767] [Other Feature Feedback, Questions, & Ideas] [answered]
Como puedo ser un co-author de un commit desde consola?.. cuales son los comandos.

**Comment by VictorHugoAguilar** [comment, marked as answer]
1. Recopila el nombre y dirección de correo electrónico de cada co-autor. Si alguien elige mantener su dirección de correo electrónico como privada, debes utilizar su correo de no-reply proporcionado por GitHub para proteger su privacidad.
2. Escriba su mensaje de confirmación y una descripción breve y significativa de sus cambios. Después de la descripción de su compromiso, en lugar de una cita de cierre, agregue dos líneas vacías.

`git commit -m "Refactor usability tests.`

3.  En la siguiente línea del mensaje de confirmación, escriba Coautor: nombre <nombre@ejemplo.com> con información específica para cada coautor. Después de la información del coautor, agregue una comilla de cierre.

Si está agregando varios coautores, asigne a cada coautor su propia línea y Co-authored-by: commit trailer.

```
$ git commit -m "Refactor usability tests.
>
>
Co-authored-by: name <name@example.com>
Co-authored-by: another-name <another-name@example.com>"
```

Referencia: [https://docs.github.com/es/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors](url)
